src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -1233,7 +1233,27 @@ public void When_Collection_Property_Nest_Multiple_Collections()
 			Assert.AreEqual(1, SUT.LeftItems.Count);
 			Assert.AreEqual("qwe", SUT.LeftItems[0].Text);
 		}
-		
+
+		[TestMethod]
+		public void When_StaticResource_In_ResourceDictionary()
+		{
+			var s = GetContent(nameof(When_StaticResource_In_ResourceDictionary));
+			var r = Windows.UI.Xaml.Markup.XamlReader.Load(s) as UserControl;
+
+			var panel = r.FindName("panel") as StackPanel;
+			Assert.IsNotNull(panel);
+
+			var c2 = (Color)panel.Resources["c2"];
+			var b2 = (SolidColorBrush)panel.Resources["b2"];
```

Comment:
It's unclear why you are doing those assignations (and without checking them)

Reply:
Will make it clearer in the next PR.